surface/src/3rdparty/opennurbs/opennurbs_bezier.cpp

```diff
@@ -1374,7 +1374,7 @@ bool ON_BezierCurve::GetCV( int i, ON::point_style style, double* Point ) const
   switch(style) {
   case ON::euclidean_rational:
     Point[dim] = w;
-    // no break here
+    // fall through
```

Comment:
We've merged a new `FALLTHROUGH` macro some days ago: https://github.com/PointCloudLibrary/pcl/pull/3320/commits/df40206d46205879caadc8f253c01db779bb6bc5, I think it will fit here nicely?